### ISSUE TITLE
[Feat] Export to Anki option

### ISSUE THREAD

**Issue by ShreyasKallingal** [st3v3nmw/obsidian-spaced-repetition#27]
This a fantastic plugin to begin with—I really appreciate your work!

Would it be possible to have an export option to an Anki-compatible APKG? Thinking about long-term viability, Anki is a lot more established when it comes to SRS, and along with that comes a suite of addons like image occlusion. The main draw to Obsidian for me is the data security and longevity associated with plain text files. I think adding an Anki export option would achieve this for the flashcards as well. 

This feature request is a bit out there, and I guess it's more of a discussion about the potential of this plugin. I would love to hear what you think about it :)

**Comment by st3v3nmw** [comment]
Hello,
Yes, this is already on the roadmap. I hadn't yet created an issue so I'll use this one instead.
Thanks for your input.

**Comment by DaveyUS** [comment]
There is already a very popular plugin that handles exactly this problem - [Obsidian_to_Anki by Pseudonium](https://github.com/Pseudonium/Obsidian_to_Anki) -  Seeing as most people will use this plugin to _get away from Anki_, I wonder if adding this feature would be a waste of work time for little gain?

**Comment by domkm** [comment]
There are actually two [Obsidian plugins for Anki](https://github.com/reuseman/flashcards-obsidian). 
I agree with @DaveyUS. Including Anki functionality in this plugin seems redundant.

**Comment by ShreyasKallingal** [comment]
I guess you're right. My issue with those is that the ID gets added to a new line, so it breaks lists. And they seem to get sparse updates, so I just thought it would be nice to have an Anki option.

**Comment by DaveyUS** [comment]
@ShreyasKallingal I feel you on that. Have you tried contacting one of those devs to try to get a work around for that issue?

**Comment by st3v3nmw** [comment]
I was thinking that the export to Anki be a one-time thing (not syncing like the other plugins do). For instance, if someone no longer wants to use this plugin they can export everything into an APKG they can import in Anki.

**Comment by DaveyUS** [comment]
Our point was simply that there are already two robust plugins that solve for the Obsidian to Anki workflow, so it might be redundant to add to your plugin which is designed to live solely inside of obsidian... however it's your plugin and we are grateful for any work you do on it!

**Comment by jsmm** [comment]
There is also an Anki plugin, [Obsidianki](https://ankiweb.net/shared/info/620260832). It imports cards without the need to install a second spaced repetition plugin in Obsidian.

**Comment by MatthewJMullins** [comment]
There might still be some relevancy of this feature request, even when considering the other options. The main reason you would add the export to .anki feature in this plugin, rather than exporting with the other plugins: 
> A user wants to retain their progress on each card and have that transfer to Anki (i.e. I wouldn't want all my "easy" cards to now be new in the schedule).

The initial reaction is this Anki export feature would be a conflict of interest. However, I'd encourage you to consider the user who specifically stays away from plugins/apps that seem to lock them in. 

> God forbid you lost interest in this plugin someday, it's nice to know you can extract your cards **and their progress** via a .anki export.

**Comment by jakeoung** [comment]
Any news on this feature? I'm looking forward..

**Comment by rubbish** [comment]
I feel like exporting to Anki really is out of scope of this plugin. I think that this plugin should really be focused on spaced repetion within Obsidian. Like mentioned, there are other plugins that do anki export and I think that probably support could get added to them instead to support the type of regexes that this plugin already does.

**Comment by jakeoung** [comment]
Each plugin has its own syntax and my markdown note will be messy if I use two different plugins. I don't think it's a good idea to use different plugins together.

**Comment by andrewcrook** [comment]
It could be useful. You wouldn't have to mix the extra syntax for two plugins which could get messy in notes. Maybe have one way syncing with Anki. Also control which cards get exported by using a tag. However, I agree that spaced repetition isn't the same as using flashcards in the traditional sense but may have some overlap?

**Comment by Comprehensive-Jason** [comment]
> This feature has become necessary since obsidian to anki is no longer maintained, and seems to have a lot of issues opened with special cases.

I agree. Pseudonium's [Obsidian to Anki](https://github.com/Pseudonium/Obsidian_to_Anki) plugin has been dead for 2 years, yet many people still use it over Reuseman's [Obsidian Flashcards](https://github.com/reuseman/flashcards-obsidian) plugin because of how powerful and customizable the former still is. Bringing its features into this plugin would make this the one-stop shop for everything.

**Comment by YureiKertia** [comment]
I would also love this feature, specifically for the flashcard format. All my flashcards are already in spaced repetition format (which are like, more than 3000 at this point) and I just recently got my kindle jailbroken and set up with KOreader. If I was able to extract a deck to .apkg, I could export them onto my Kindle and use it to study my flashcards. 
Fully understand if this isn't high priority though, love your work anyways!
